docs/doc/13-sql-reference/10-data-types/index.md

```diff
@@ -7,36 +7,37 @@ slug: ./
+| TINYINT   | INT8    | 1 byte       | -128                     | 127                            |                                                                         |
+| SMALLINT  | INT16   | 2 bytes      | -32768                   | 32767                          |                                                                         |
+| INT       | INT32   | 4 bytes      | -2147483648              | 2147483647                     |                                                                         |
+| BIGINT    | INT64   | 8 bytes      | -9223372036854775808     | 9223372036854775807            |                                                                         |
+| FLOAT     |         | 4 bytes      | -3.40282347e+38          | 3.40282347e+38                 |                                                                         |
+| DOUBLE    |         | 8 bytes      | -1.7976931348623157E+308 | 1.7976931348623157E+308        |                                                                         |
+| DECIMAL   |         | 16 bytes     |                          |                                |                                                                         |
+| DATE      |         | 4 bytes      | 1000-01-01               | 9999-12-31                     | YYYY-MM-DD                                                              |
+| TIMESTAMP |         | 8 bytes      | 0001-01-01 00:00:00      | 9999-12-31 23:59:59.999999 UTC | YYYY-MM-DD hh:mm:ss[.fraction], up to microseconds (6 digits) precision |
+| VARCHAR   | STRING  | variable     |                          |                                |                                                                         |
 
 ## Semi-structured Data Types
 
 Databend supports three Semi-structured types: ARRAY, OBJECT and VARIANT.
 
-| Name    |    Aliases   | Build From Values       | Description
-|---------|--------------|-------------------------|----------------
-| ARRAY   |              | [1,2,3]                 | Zero-based indexed list, each value can have difference data type.
-| TUPLE   |              | ('2023-02-14 08:00:00','Valentine's Day') | Collection of ordered,immmutable, which requires the type of each element to be declared before being used.
-| VARIANT |  JSON        | [1,{"a":1,"b":{"c":2}}] | Collection of elements of different data types., including ARRAY and OBJECT.
+| Name    | Aliases | Build From Values                         | Description                                                                                                 |
+|---------|---------|-------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| ARRAY   |         | [1,2,3]                                   | Zero-based indexed list, each value can have difference data type.                                          |
```

Comment:
VARIANT is the only SEMI struct datatype. `OBJECT` should be removed in the docs, let's use `VARIANT` instead.

Reply:
```
Zero-based indexed list, each value can have difference data type.
```

It's wrong,  we have `Array<T>` . It's inner type is fixed.

Reply:
Ok, let me remove it.

Reply:
Also `Tuple` is not semi struct datatype, it's a nested datatype.